planet/clients/subscriptions.py

```diff
@@ -95,7 +95,8 @@ async def list_subscriptions(self,
             dict: a description of a subscription.
 
         Raises:
-            ClientError
+            APIError: on an API server error.
+            ClientError: on a client error.
```

Comment:
@jreiberkyle thanks for pointing out that we intend to raise APIError when a server-side error happens. I've documented that we intentionally raise two kinds of error.

Reply:
We don't have any code yet that would raise ClientError, but soon enough!

Reply:
Thank goodness for documentation is all I can say. I keep having to go back to remind / reorient myself.